packages/firestore/tools/console.build.js

```diff
@@ -19,34 +19,44 @@
  * Firebase console uses firestore in its special way.
  * This file creates a build target for it.
  */
+const tmp = require('tmp');
 const rollup = require('rollup');
-const typescriptPlugin = require('rollup-plugin-typescript2');
-const typescript = require('typescript');
-const resolve = require('rollup-plugin-node-resolve');
-const { uglify } = require('rollup-plugin-uglify');
 const fs = require('fs');
 const util = require('util');
 const fs_writeFile = util.promisify(fs.writeFile);
 
-const plugins = [
-  resolve(),
-  typescriptPlugin({
-    typescript
-  }),
-  uglify({
-    output: {
-      ascii_only: true // escape unicode chars
-    }
-  })
-];
+const rollupUtil = require('../rollup.shared');
 
 const EXPORTNAME = '__firestore_exports__';
 
-const inputOptions = {
+const tmpFile = tmp.fileSync().name;
+
+const es2017InputOptions = {
   input: 'index.console.ts',
-  plugins
+  // If I set mangled to true the build breaks, but all other build pipelines
```

Comment:
The problem seems to come from the 2 interface exports - `FirestoreDatabase` and `FirstPartyCredentialsSettings`. Once they are removed, the mangling works again.

The issue seems to be that they are not included in the public externs.

Reply:
I don't think the interfaces provide any value here since console uses the js output, so they can probably be removed.